packages/vision/src/index.js

```diff
@@ -519,6 +535,7 @@ Vision.prototype.detect = function(images, options, callback) {
     });
   });
 };
+// jscs:enable maximumLineLength
```

Comment:
I had to put this below the function itself (which I know is undesirable) because dox was flipping out.